starboard/shared/win32/system_get_device_type.cc

```diff
@@ -1,28 +0,0 @@
-// Copyright 2017 The Cobalt Authors. All Rights Reserved.
```

Comment:
The entire win32 directory can completely go at this point.

Reply:
Does https://github.com/youtube/cobalt/pull/4119 take care of that?